fuse_publishers/test/test_pose_2d_publisher.cpp

```diff
@@ -232,48 +251,53 @@ TEST_F(Pose2DPublisherTestFixture, PublishPose)
 TEST_F(Pose2DPublisherTestFixture, PublishPoseWithCovariance)
 {
   // Test that the expected PoseWithCovarianceStamped message is published
+  auto node = rclcpp::Node::make_shared("test_pose_2d_publisher_node");
+  executor_->add_node(node);
+
+  fuse_core::getParam(node, "map_frame", std::string("test_map"));
```

Comment:
Recommend a comment that these calls are declaring parameters on the node, which Pose2DPublisher will then use.

Alternatively, could pass a `NodeOptions` instance with [arguments](https://github.com/ros2/rclcpp/blob/3fb012e2e979475f5044ab0e0f9b91d336db5f46/rclcpp/include/rclcpp/node_options.hpp#L117) set to set these values. I think that would be closer to actual usage.

https://docs.ros.org/en/rolling/How-To-Guides/Node-arguments.html

```C++
rclcpp::NodeOptions options;
options.arguments({
 "--ros-args",
 "-p", "map_frame:=test_map",
 "-p", "odom_frame:=test_odom",
 "-p", "base_frame:=test_base",
 "-p", "publish_to_tf:=false});
```

Reply:
https://github.com/locusrobotics/fuse/compare/f9d587a8c481bab1398f71dd2a4a5eb28fa82547..6ee89e8d6b24b282fb99c14482fe2d6412dcc3e9